predicators/spot_utils/integration_tests.py

```diff
@@ -0,0 +1,191 @@
+"""Integration tests for spot utilities.
+
+Run with --spot_robot_ip and any other flags.
```

Comment:
i see and like what you're doing here, but I can't help but wonder if there's some way to make it more general. I.e. - if we try to do Spot stuff at LIS or in a different room - how easy will it be to configure these tests? It seems like we just need to mess with the offsets, etc. a bit - which is pretty nice! But is there some way we can write these tests so that we can basically just run them in any new env we make? (Probably something we can table for later when we actually have the need to do this, but worth thinking about i think)

Reply:
Yeah good question. I think there are two main blockers toward making the test more general:
(1) The offset thing, which seems to be to require some genuine TAMP-like sampling. The issue is that the tables have walls on certain sides which prevent navigating to certain pre-pick positions.
(2) The rotation of the table with respect to the april tag is not known (because we're only getting x, y, z from the april tag). Since we want to place a little bit "behind" the april tag, i.e. on the table itself, we need to effectively hard-code this rotation (or the offset to the origin w.r.t. the april tag).

Both these seem like surmountable issues but I think would require more work than worth it right now?